packages/table/src/table2.tsx

```diff
@@ -669,11 +669,15 @@ export class Table2 extends AbstractComponent2<Table2Props, TableState, TableSna
             return false;
         }
 
-        const rowIndices = this.grid.getRowIndicesInRect(viewportRect, enableGhostCells!);
+        const rowIndices = this.grid.getRowIndicesInRect(
+            viewportRect,
+            enableGhostCells!,
+            this.columnHeaderHeight || Grid.MIN_COLUMN_HEADER_HEIGHT,
```

Comment:
used `||` vs `??` bc `this.columnHeaderHeight` was inconsistently being `0` vs the actual height when running tests

Reply:
ok, that's fair, but I think I'd rather not make an exception to the [strict-boolean-expressions](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md) rule here (we don't actually lint for it, but I try to enforce it manually). I find the `||` hard to read, how about something like this instead:

```suggestion
            this.columnHeaderHeight > 0 ? this.columnHeaderHeight : Grid.MIN_COLUMN_HEADER_HEIGHT,
```

Reply:
actually, on second thought... we never really want `this.columnHeaderHeight` to be `0`.

we could change the ref handler to prevent that from happening, and keep the min-height logic scoped to that part of this component's code:

```ts
private refHandlers = {
        columnHeader: (ref: HTMLElement | null) => {
            this.columnHeaderElement = ref;
            if (ref != null) {
                this.columnHeaderHeight = Math.max(ref.clientHeight, Grid.MIN_COLUMN_HEADER_HEIGHT);
            }
```